app/views/user/show.html.erb

```diff
@@ -1 +1,10 @@
-<h1>Show users</h1>
\ No newline at end of file
+<section class="row">
+  <div class="col-12">
+  <%= render partial:'partials/user', locals: {user: @user} %>
+  </div>
+  <div class="col-12 border border-dark my-2 mt-4">
+    <h4>Bio</h4>
+    <p><%= @user.bio %></p>
+  </div>
+  
```

Comment:
- [ ] It would be better if we display the 3 most recent posts from the user. Let us kindly check the wireframe and make the app accordingly

Reply:
so I will repeat a week because of this? they are no posts yet. create a post is in the next project